pkg/engine/handlers/validation/validate_cel.go

```diff
@@ -140,7 +142,7 @@ func (h validateCELHandler) Process(
 		ns = ""
 	}
 	if ns != "" {
-		if h.client != nil {
+		if h.client != nil && h.isCluster {
```

Comment:
Why does `isCluster` matter? The `client` is not nil indicating it has "access" to the cluster right?

Reply:
It natters because the client is now never nil, when offline then the client is a fake client, so another way to check if its a cluster operation is needed. the nil check is kept for safety purposes